mappers/contact-info.js

```diff
@@ -170,7 +170,7 @@ function mapOfferingAgent(recordId, record, postfix = '') {
     if (title) {
       contactStatements.push(new Statement(sym(contactPointUri), VCARD('honorific-prefix'), lit(title, 'nl')));
     } else {
-      console.error(`Cannot map agent title value '${record[`agent_title${postfix}`]}' for record ${recordId}`);
+      errorLogger('agent title' , record[`agent_title${postfix}`], recordId);
```

Comment:
Pass the same field name (`agent_title${postfix}`) as used to lookup the value as first argument. I know it was different before, but old code is wrong.